diff --git a/hmac.c b/hmac.c
index fa2b689..198c1e5 100644
--- a/hmac.c
+++ b/hmac.c
@@ -59,7 +59,7 @@ void hmac_sha256_Final(HMAC_SHA256_CTX *hctx, uint8_t *hmac) {
   memzero(hctx, sizeof(HMAC_SHA256_CTX));
 }
 
-void hmac_sha256(const uint8_t *key, const uint32_t keylen, const uint8_t *msg,
+void trzr_hmac_sha256(const uint8_t *key, const uint32_t keylen, const uint8_t *msg,
                  const uint32_t msglen, uint8_t *hmac) {
   static CONFIDENTIAL HMAC_SHA256_CTX hctx;
   hmac_sha256_Init(&hctx, key, keylen);
diff --git a/hmac.h b/hmac.h
index 3921a17..30052eb 100644
--- a/hmac.h
+++ b/hmac.h
@@ -42,7 +42,7 @@ void hmac_sha256_Init(HMAC_SHA256_CTX *hctx, const uint8_t *key,
 void hmac_sha256_Update(HMAC_SHA256_CTX *hctx, const uint8_t *msg,
                         const uint32_t msglen);
 void hmac_sha256_Final(HMAC_SHA256_CTX *hctx, uint8_t *hmac);
-void hmac_sha256(const uint8_t *key, const uint32_t keylen, const uint8_t *msg,
+void trzr_hmac_sha256(const uint8_t *key, const uint32_t keylen, const uint8_t *msg,
                  const uint32_t msglen, uint8_t *hmac);
 void hmac_sha256_prepare(const uint8_t *key, const uint32_t keylen,
                          uint32_t *opad_digest, uint32_t *ipad_digest);
